### ISSUE TITLE
plugin does not work with + iOS 11

### ISSUE THREAD

**Issue by rcasunshare** [ohh2ahh/AppAvailability#65]
This plugin does not work with + iOS 11

On Android you have a terrible blunder with Facebook

**Comment by fdhdsjkfhds** [comment]
same but trying for IOS 13

**Comment by pke** [comment]
What exactly is the problem?

**Comment by jeffmitaly** [comment]
Yes please specify the exact problem you are seeing. This issue made me think this plugin was not working but it actually is working fine on iOS 14 .

In my case, I had not updated my <key>LSApplicationQueriesSchemes</key> properly in Info.plist so it appeared the plugin could never determined if the app I was targeting was installed or not.

**Comment by ahaneef29** [comment]
**Fixes**
After Updating the config.xml file. if you rebuild project iOS plist is not reflecting.  Try removing  the platform or manually add the schemes in the LSApplicationQueriesSchemes array.

` <config-file parent="LSApplicationQueriesSchemes" target="*-Info.plist">
            <array>
                <string>abc</string>             
            </array>
        </config-file>`

**Comment by pke** [comment]
So this can be closed then @rcasunshare when @ahaneef29 suggestions work?
